guides/mentorship/README.md

```diff
@@ -0,0 +1,33 @@
+#Made Tech Mentorship Guide
+
+##Overview
+At Made Tech, mentorship enables Made Tech staff with the opportunity to provide guidance and support to people in the wider tech community. Mentees are able to gain new knowledge and support and develop and enhance their skills.
+
+###General Guidance
```

Comment:
It looks like the level 3 headers here should all be level 2.

Similarly in other files, I think. Not sure it makes sense hierarchically. :)